code/modules/clothing/head/costume.dm

```diff
@@ -192,3 +192,10 @@
 	name = "allies helmet"
 	icon_state = "allies_helmet"
 	inhand_icon_state = "allies_helmet"
+
+/obj/item/clothing/head/nun // Alternative version of old nun headdress with a lot more detail. Added 2022-08, paired with Nun robes
```

Comment:
this isn't modular

Reply:
What change exactly are you suggesting? Another item path? Putting it in a new dm? Please elaborate.

Reply:
Follow the guide from the [readme](https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md)